### PR TITLE
Update CodeGeneration

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

File: Sources/SwiftSyntax/generated/Tokens.swift [hunk 1]
```diff
@@ -14,713 +14,713 @@
...
-        .prefixOperator(text), 
-        leadingTrivia: leadingTrivia, 
-        trailingTrivia: trailingTrivia, 
-        presence: presence
-      
+      .prefixOperator(text),
+      leadingTrivia: leadingTrivia,
+      trailingTrivia: trailingTrivia,
+      presence: presence
+
```
Comment (kimdv) [inline Sources/SwiftSyntax/generated/Tokens.swift:520]: There are multiple places with with white space.
Also think that Doug mentioned it.

**Review by kimdv — APPROVED** [review]

File: Sources/SwiftSyntax/generated/Tokens.swift (hunk 1 above)
Comment (bnbarham) [reply Sources/SwiftSyntax/generated/Tokens.swift:520]: @ahoppen this one seems pretty weird to me:
1. I thought we were stripping trailing whitespace
2. We shouldn't be adding a newline here. Seems like that was broken before and after though

**Comment by kimdv** [comment]
@ahoppen how much white space did we remove? 😅😆

File: Sources/SwiftSyntax/generated/Tokens.swift (hunk 1 above)
Comment (ahoppen) [reply Sources/SwiftSyntax/generated/Tokens.swift:520]: This is the formatter subclass of `BasicFormat` that lives in CodeGeneration messing things up.

**Pull request merged**